Reqnroll/EnvironmentAccess/BuildMetadata.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Reqnroll.EnvironmentAccess
+{
+    public class BuildMetadata
```

Comment:
Is public needed for the classes and interfaces?

Reply:
I could be wrong, but this feels like an area that will want a plug-in architecture to match build systems, where each plug-in knows how to access the information of a specific system. If this is the case, these types should be public.

Reply:
OK! :)